### PR TITLE
devbot improvements

### PR DESCRIPTION
**Pull request by TAG-Epic**
- shows devbot on ls list

### CODE REVIEW

**Comment by quackduck** [comment]
~~this one is easy to review~~

**Comment by quackduck** [comment]
“Oops”

File: commands.go [hunk 1]
```diff
@@ -531,6 +531,16 @@ func banCMD(line string, u *User) {
 	var victim *User
 	var ok bool
 	banner := u.Name
+
+    boreName := "http://bore.pub:1337"
+    if split[0] == boreName {
+        bore, boreOnline := findUserByName(u.room, boreName)
+        if boreOnline {
+            u.room.broadcast(bore.Name, "Nice try devbot.")
+            u.room.broadcast("", "devbot has been banned by " + banner)
```
Comment (quackduck) [inline commands.go:540]: You do know this won't actually disable devbot right.

**Comment by quackduck** [comment]
What's the rationale for adding devbot to the user list?

File: commands.go (hunk 1 above)
Comment (TAG-Epic) [reply commands.go:540]: Yes

**Comment by quackduck** [comment]
Also, I don't think it would be good idea for me to introduce this special case for _all_ instances of Devzat. I might allow plugins that treat a username in a special way to run on the main server though.

**Comment by TAG-Epic** [comment]
> What's the rationale for adding devbot to the user list?

Devbot is always here, he should be considered a user

File: commands.go [hunk 2]
```diff
@@ -777,6 +787,7 @@ func lsCMD(rest string, u *User) {
 		for _, us := range u.room.users {
 			s += us.id + " " + us.Name + "  \n"
 		}
+        s += strings.Repeat("0", 64) + Devbot + " \n"
```
Comment (quackduck) [inline commands.go:790]: Adding devbot to the ID list isn't useful.
```suggestion
```

**Comment by quackduck** [comment]
Okay let's try it

**Pull request merged**